roles/openshift_node/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+# tasks file for openshift_node
+- name: Install OpenShift
+  yum: pkg=openshift-node state=installed
+
+  # fixme: Once openshift stops resolving hostnames for node queries remove this...
+- name: Set hostname to IP Addr (WORKAROUND)
+  command: /usr/bin/hostname {{ oo_bind_ip }}
+
+- name: Retrieve OpenShift Master credentials
+  local_action: command /usr/bin/rsync --compress --archive --rsh 'ssh  -S none -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' root@{{ oo_master_public_ips[0] }}:/var/lib/openshift/openshift.local.certificates/admin/ {{ tmp_dir }}
+  ignore_errors: yes
```

Comment:
I just finished implimenting this same thing. Was hoping for better support for this type of task in ansible.